README.md

```diff
@@ -118,7 +118,10 @@ const perfume = new Perfume({
         myAnalyticsTool.track('storageEstimate', data);
         break;
       case 'TTFB':
-        myAnalyticsTool.track('timeToFirstByte', { duration: data });
+        // Zero value mean than page restored from Back-Forward Cache, and you can ignore it before reporting
+        if (data > 0) {
```

Comment:
I feel like this check should be part of Perfume at this point, no?

Reply:
I don't follow, what do you mean?

Reply:
Just don't have an answer, is `0` values a problem or not, and users must decide it or perfume.js internally

Because `web-vitals` still report this values with `navigationType: 'back-forward-cache'`

Reply:
Maybe ignoring `0` is a really useful default behaviour

Reply:
Yeah, if we don't, 50% of people will forget to read the docs, and they will get like a wrong Good Score.

Let's do this, let's avoid reporting `0`, after all the end users doesn't really care how many time there was a `'back-forward-cache'`. Is not something a User can really do much about it.

Reply:
Good point) Will rework soon

Reply:
Zero value may mean it's 0, or it's not set (Safari reports zero for cross-origin redirect for example).

For this uncertainty reason we choose NOT to emit 0 TTFBs at all. Technically we could do it for bfcache restores (as it's by definition not a redirect and really is known to be zero), and we debated that, but in the end decided to not emit there either for consistency.

Reply:
Sent fix where ignore zero

Reply:
As of this change we don't "report" a 0 TTFB: https://github.com/GoogleChrome/web-vitals/pull/281/files (see also the comments in that PR).